src/core/include/openvino/op/squeeze.hpp

```diff
@@ -41,11 +41,12 @@ class OPENVINO_API Squeeze : public Op {
 }  // namespace v0
 
 namespace v15 {
-class OPENVINO_API Squeeze : public v0::Squeeze {
+class OPENVINO_API Squeeze : public Op {
```

Comment:
There can be Squeeze base class to not duplicate common parts of v0 and v15

Reply:
Sure, I will add it.